cmd/slinky/config.go

```diff
@@ -0,0 +1,147 @@
+package main
+
+import (
+	"fmt"
+	"github.com/skip-mev/slinky/providers"
+	"github.com/spf13/viper"
+	"time"
+
+	"github.com/skip-mev/slinky/oracle/config"
+)
+
+const (
+	// DefaultUpdateInterval is the default value for how frequently slinky updates aggregate price responses.
+	DefaultUpdateInterval = 250000000
+	// DefaultMaxPriceAge is the default value for the oldest price considered in an aggregate price response by slinky.
+	DefaultMaxPriceAge = 120000000000
+	// DefaultPrometheusServerAddress is the default value for the prometheus server address in slinky.
+	DefaultPrometheusServerAddress = "0.0.0.0:8002"
+	// DefaultMetricsEnabled is the default value for enabling prometheus metrics in slinky.
+	DefaultMetricsEnabled = true
+	// DefaultHost is the default for the slinky oracle server host.
+	DefaultHost = "0.0.0.0"
+	// DefaultPort is the default for the slinky oracle server port.
+	DefaultPort = "8080"
+)
+
+type OracleConfig struct {
```

Comment:
Is this idea for us to have the `LegacyOracleConfig` be a sidecar thing, and this `OracleConfig` be a separate config?

Reply:
If so, I'd prefer that we canonicalize this version of the config + handle the adaptation between legacy config / this one during reading the legacy config from file?